docs/setup-dev-environment.md

```diff
@@ -1,43 +1,116 @@
 # Setting up a local development environment
 
-This document is intended to guide you through setting up a local development environment for
-Colouring London. This guide assumes you already have Ubuntu 20.04 server installed, typically
-installed in a virtual environment such a Virtual Box and are able to SSH into your Ubuntu
-installation for convenience.
+This document is intended to guide you through setting up a local development environment for the Colouring London application. This guide assumes you already have either already have access to an machine with Ubuntu 18.04 or 20.04 installed, or can use VirtualBox to set up an Ubuntu virtual machine as below:
```

Comment:
Is it also supposed to work on 20.04? Or at least beneficial to test with 20.04?

Reply:
I tested on 18.04 simply because I forgot to apply the update to 20 when I created my VM in Virtualbox - however I assume it works on 20 if that is what you have used? I could test this further to be sure, what do you think?

Reply:
I plan it on testing anyway, so I can do this and let you know if anything breaks (and retest on 18.04 in such case)

Reply:
ok great thanks